src/locales/hu.yaml

```diff
@@ -1,508 +1,499 @@
+    hint:
+      add_printer: 'Pl.: Http://fluiddpi.local'
+    msg:
+      trouble: >-
+        Problémád akadt? További információt <a href="%{url}"
+        target="_blank">itt találsz</a> .
+    tooltip:
+      endpoint_examples: >-
+        Írd be az API URL-jét.<br />Néhány példa:<br
+        /><blockquote>http://fluidd.local, http://192.168.1.150</blockquote>
+  endstop:
+    label:
+      open: NYITVA
+      triggered: AKTIVÁLVA
+    msg:
+      subtitle: Az endstop állapotának frissítéséhez használd a Frissítés gombot.
+  file_system:
+    filters:
+      label:
+        print_start_time: Nyomtatottak szűrése
+        print_start_time_desc: Kiszűri a már kinyomtatott elemeket.
+    label:
+      dir_name: Könyvtár neve
+      disk_usage: Lemezhasználat
+      diskinfo: Lemezinformációk
+      downloaded: Letöltve
+      file_name: Fájlnév
+      transfer_rate: Átviteli sebesség
+      uploaded: Feltöltve
+    msg:
+      confirm: Biztos vagy benne? Ez ki fogja törölni az összes fájlt és mappát.
+      not_found: Nem találhatóak fájlok
+      processing: Feldolgozás
+    overlay:
+      label: Húzza <strong>ide</strong> a fájlt
+    title:
+      add_dir: Könyvtár hozzáadása
+      add_file: Fájl hozzáadása
+      download_file: Fájl lekérése
+      keyboard_shortcuts: Gyorsbillentyűk
+      rename_dir: Címtár átnevezése
+      rename_file: Fájl átnevezése
+      upload_file: Fájl feltöltése
+    tooltip:
+      low_on_space: Kevés a lemezterület
+      root_disabled: '{root} root nem érhető el. Kérjük, ellenőrizze a naplóit.'
+  gcode:
+    btn:
+      load_current_file: Az aktuális fájl betöltése
+    label:
+      current_layer_height: Jelenlegi rétegmagasság
+      follow_progress: Folyamat nyomonkövetése
+      layer: Réteg
+      layers: Rétegek
+      parsed: Elemezve
+      show_extrusions: Extrudálások megjelenítése
+      show_moves: Mozgások megjelenítése
+      show_next_layer: Következő réteg megjelenítése
+      show_previous_layer: Előző réteg megjelenítése
+      show_retractions: Visszahúzások megjelenítése
+    msg:
+      confirm: >-
+        A fájl "%{filename}" mérete %{size}, megterhelheti a rendszert.
+        Biztos, hogy folytatod?
+  general:
+    btn:
+      add: Hozzáad
+      add_dir: Könyvtár hozzáadása
+      add_file: Fájl hozzáadása
+      add_printer: Nyomtató hozzáadása
+      adjust_layout: Irányítópult elrendezésének beállítása
+      all: Mindegyik
+      calibrate: Kalibrálás
+      cancel: Mégsem
+      clear_profile: Profil törlése
+      close: Bezárás
+      config_reference: Konfigurációs hivatkozás
+      download: Letöltés
+      edit: Szerkesztés
+      exit_layout: Kilépés az elrendezés módból
+      extrude: Extrudálás
+      heaters_off: Fűtések KI
+      load_all: Az összes betöltése
+      more_information: Több információ
+      pause: Szünet
+      preheat: Előfűtés
+      presets: Előbeállítások
+      preview_gcode: A G-Kód előnézete

```

Comment:
Up to you on this one, but normally I wouldn't expect for "G-Code" to be translated as it is a specific technical term/format.

Reply:
I think having this as "G-Code" makes sense, the new chinese translation PR also uses the english version and IIRC so does the german translation